assets/stylesheets/hashicorp/enterprise-badges.scss

```diff
@@ -0,0 +1,55 @@
+$label-enterprise-font-size: 12px;
+$label-enterprise-font-size-lg: 16px;
+$label-enterprise-alignment-tweak: 1px; //tweak font not aligning vertically centered
+
+$header-font-family: 'klavika-web', Helvetica, sans-serif;
+
+.label-enterprise {
```

Comment:
There seems to be some tabs-v-spaces going on here. Can you please make sure you're only using spaces in the CSS files?